backend/routes/templates.go

```diff
@@ -93,7 +93,12 @@ func addTemplateData(w http.ResponseWriter, r *http.Request) {
 }
 
 func addTemplateHashDevnet(w http.ResponseWriter, r *http.Request) {
-	// TODO: Disable this in production
+	// Disable this in production
+	if core.ArtPeaceBackend.BackendConfig.Production {
```

Comment:
This is good, but could you also disable the respective routes in the `Init*Routes` functions so they are not callable?

Reply:
Ok, handles for `/addTemplateHashDevnet` and `/placePixelDevnet` are enabled only when production flag is false.